corehq/util/workbook_json/excel.py

```diff
@@ -190,7 +190,7 @@ def _convert_float(value):
 class WorkbookJSONReader(object):
 
     def __init__(self, file_or_filename):
-        if isinstance(file_or_filename, InMemoryUploadedFile):
+        if isinstance(file_or_filename, (InMemoryUploadedFile, file)):
```

Comment:
This is necessary because the name the tempfile name ```file_or_filename.name``` is ```<fdopen>```, which openpyxl is unable to handle.